overrides/resources/betterquesting/lang/en_us.lang

```diff
@@ -1185,7 +1185,7 @@ dj2.quest.db.561.desc=Craft a full set of Chaotic Armor. They're an even better
 dj2.quest.db.562.title=It's pure Chaos
 dj2.quest.db.562.desc=Craft a Chaotic Staff of Power.%n%n§c§lWARNING:§r Before you use this tool, you'll need to stabilize it. Read the Draconic Information Tablet -> Draconic Additions -> Chaos Stabilizer section on how to perform the Stabilization Ritual.
 dj2.quest.db.563.title=Imagine if time passed faster...
-dj2.quest.db.563.desc=Craft an Imaginary Time Block! This machine speeds up the tickrate of nearby blocks, making it ideal for a large crop farm. It just requires power and nothing else!
+dj2.quest.db.563.desc=Craft an Imaginary Time Block! This machine speeds up the tickrate of nearby blocks, making it ideal for a large crop farm. It just requires power and nothing else!%n%nIts radius is 3 blocks horizontal and 2 blocks vertical, making it a 7x5x7 area.
```

Comment:
```suggestion
dj2.quest.db.563.desc=Craft an Imaginary Time Block! This machine speeds up the tickrate of nearby blocks, making it ideal for a large crop farm. It just requires power and nothing else!%n%nIt affects nearby blocks 3 blocks horizontally and 2 blocks vertically, forming a 7x5x7 area around the Imaginary Time Block.
```
Might be a little better? This might be better as a tooltip though.

Reply:
Yeah, tbh I don't really care about the wording, I just wanna convey the info, you can make the sentences prettier if you want. And yes it could also be a tooltip but I'd rather have all the info in the quest than make tooltips for many items. I don't wanna look at a block of text when hovering over an item and quests should already tell people the important infos about a mod/item/block.